ext/mysqli/tests/deprecated_constants.phpt

```diff
@@ -0,0 +1,38 @@
+--TEST--
+Deprecated constants
+--SKIPIF--
+<?php
+require_once('skipif.inc');
+?>
+--FILE--
+<?php
+
+echo constant('MYSQLI_NO_DATA')."\n";
+echo constant('MYSQLI_DATA_TRUNCATED')."\n";
+echo constant('MYSQLI_SERVER_QUERY_NO_GOOD_INDEX_USED')."\n";
+echo constant('MYSQLI_SERVER_QUERY_NO_INDEX_USED')."\n";
+echo constant('MYSQLI_SERVER_QUERY_WAS_SLOW')."\n";
+echo constant('MYSQLI_SERVER_PS_OUT_PARAMS')."\n";
+
+print "done!";
+?>
+--EXPECTF--
+
+Deprecated: Constant MYSQLI_NO_DATA is deprecated in %s
+100
+
+Deprecated: Constant MYSQLI_DATA_TRUNCATED is deprecated in %s
+101
+
+Deprecated: Constant MYSQLI_SERVER_QUERY_NO_GOOD_INDEX_USED is deprecated in %s
+16
+
+Deprecated: Constant MYSQLI_SERVER_QUERY_NO_INDEX_USED is deprecated in %s
+32
+
+Deprecated: Constant MYSQLI_SERVER_QUERY_WAS_SLOW is deprecated in %s
+2048
+
+Deprecated: Constant MYSQLI_SERVER_PS_OUT_PARAMS is deprecated in %s
+4096
```

Comment:
I'd avoid testing specific values here -- possibly these vary by client version.

Reply:
I added %i instead. Is this better?